kufar/books/main.go

```diff
@@ -248,54 +248,61 @@ func (d Database) SignInUser(w http.ResponseWriter, r *http.Request) {
 }
 
 //CheckAuth ...
-func (d Database) CheckAuth(w http.ResponseWriter, r *http.Request, userid int) bool {
+func (d Database) CheckAuth(w http.ResponseWriter, r *http.Request) (id int, valid bool) {
```

Comment:
make my `checkAuth` func to return the userID, so even though the credentials are valid, only the owner with that bookID can update the book